go.mod

```diff
@@ -1,13 +1,15 @@
 module github.com/iotexproject/iotex-proto
 
 require (
-	github.com/golang/mock v1.4.4
-	github.com/golang/protobuf v1.4.3
+	github.com/golang/mock v1.5.0
+	github.com/golang/protobuf v1.5.2
```

Comment:
can you run `go mod tidy`?
i got v1.5.0 when running tidy

Reply:
also, can you make sure that with change in this PR
core's https://github.com/iotexproject/iotex-core/pull/2693 test can pass?

Reply:
> can you run `go mod tidy`?
> i got v1.5.0 when running tidy


I got
```
github.com/golang/mock v1.5.0	
github.com/golang/protobuf v1.5.2
```